### PR TITLE
Homework 00

### PR DESCRIPTION
**Pull request by ililikakis**
first PR

### CODE REVIEW

File: homework_00/Main.java
Comment (LScheib) [inline homework_00/Main.java:1]: Are the blank spaces here intentional?

Comment (ililikakis) [reply homework_00/Main.java:1]: Which ones?

File: homework_00/Main.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package homework_00;
+
+public class Main {
+    public static void main( String [] args) {
+        System .out. println (" Hello world !");
```
Comment (LScheib) [inline homework_00/Main.java:5]: I meant these blank spaces in the print statement.

Comment (ililikakis) [reply homework_00/Main.java]: No!
Fixed it :)

**Pull request merged**